all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -73,6 +73,26 @@ items:
 - apiVersion: v1
   kind: Service
   metadata:
+    name: jaeger-collector
```

Comment:
I like this new service, specially because it brings the all-in-one closer to the production one, but why is this needed?